app/views/projects/show.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1><%= @project.name %></h1>
+<p>Material: <%= @project.material %></p>
+<p>Challenge Theme: <%= @project.challenge.theme %></p>
+<p>Number of Contestants: <%= @project.contestants.count %></p>
```

Comment:
This works just fine and isn't really a big MVC infraction, but what I would do here is encapsulate the contestant counting into a model method in the Project model. A very minor change that just keeps any data calculation (even though this is a v basic calculation) close to the db. This is especially helpful if this were to get any more complex, i.e. if we only counted certain projects or had additional conditions attached to the calculation.